NCARDDatabase/settings.py

```diff
@@ -17,7 +17,7 @@
 from datetime import timedelta
 
 AUTO_LOGOUT = {
-    'IDLE_TIME': timedelta(seconds=5),
+    'IDLE_TIME': timedelta(minutes=10),
```

Comment:
Is 10 minutes the right idle time, or should it be longer? Maybe check with Alexia